### DISCUSSION TITLE
Release 0.20

### DISCUSSION THREAD

**Discussion by viktormalik** [bpftrace/bpftrace#2927] [General]
It's been more than 3 months since we did the last release. There's quite a lot of new unreleased functionality so I think we should release soon.

Any particular issues/PRs that need to be resolved before? I have this list so far:
- #2913 (although I'm still not sure what's the best way to take there)
- #2926 
- Maybe also #2815? Or do we let it be unreleased for a while to catch potential problems?

@danobi @ajor @fbs @jordalgo

**Comment by jordalgo** [comment]
There is no urgency in getting https://github.com/iovisor/bpftrace/pull/2815 released but that being said I think I addressed all the concerns/comments and think it's fairly safe (aside from coalescing on the right config names) so I'll leave it up to the maintainers to decide :)
